lib/plugins/delete-unused-entries.js

```diff
@@ -11,19 +11,22 @@
 
 const DeleteUnusedEntriesJSPlugin = require('../webpack/delete-unused-entries-js-plugin');
 const PluginPriorities = require('./plugin-priorities');
+const copyEntryTmpName = require('../utils/copyEntryTmpName');
 
 /**
  * @param {Array} plugins
  * @param {WebpackConfig} webpackConfig
  * @return {void}
  */
 module.exports = function(plugins, webpackConfig) {
+    const entries = [... webpackConfig.styleEntries.keys()];
+
+    if (webpackConfig.copyFilesConfigs.length > 0) {
+        entries.push(copyEntryTmpName);
+    }
```

Comment:
Not a super important detail, but I wonder if this will play nice with `entrypoints.json` - i.e. not have this fake entry there after #410. I suppose our "shared entry" already isn't a problem? So we'd be good here

Reply:
Yep, I started the PR before #410 was created.
A check will have to be added [there](https://github.com/symfony/webpack-encore/pull/410/files#diff-cfa2bee523e3ea72d0447f6181fa3d5aR18) to exclude the temp file.